peepingtom/utils/components.py

```diff
@@ -1,22 +1,47 @@
-"""
-Basic data structures and base
-"""
+# Basic data structures
```

Comment:
I would leave module docstrings as they were (fixing the text obviously). If we ever decide to do some automated docs, these are recognized properly, and in general python knows what they are, as opposed to simple comments.